stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -153,8 +153,14 @@ fun TextField(
 
     LaunchedEffect(fieldState) {
         // When field is in focus and full, move to next field so the user can keep typing
+        @Suppress("SwallowedException")
         if (fieldState == TextFieldStateConstants.Valid.Full && hasFocus.value) {
-            focusManager.moveFocus(nextFocusDirection)
+            try {
+                focusManager.moveFocus(nextFocusDirection)
+            } catch (e: IllegalArgumentException) {
```

Comment:
Should we log this? It seems like it would be useful to help debug what's going wrong when the focus doesn't move.

Reply:
good idea, updated